observations/README.md

```diff
@@ -0,0 +1,1279 @@
+Additionally, any `ObservationStatement` component will be mapped to a separate `Observation` and referenced using the 
+`"derived-from"` / `"has-member"` relationship.
+
+### Componentised observation header (XML HL7 > JSON FHIR)
+
+| Mapped to (JSON FHIR Observation resource field) | Mapped from (XML HL7 / other)                                                                                            |
+|--------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| id                                               | `CompoundStatement / id [@root]`                                                                                         |
+| meta.profile\[0]                                 | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Observation-1"`                             |
+| identifier\[0].system                            | `"https://PSSAdaptor/{{losingOdsCode}}"` - where the `{{losingOdsCode}}` is the ODS code of the losing practice          |
+| identifier\[0].value                             | `CompoundStatement / id \[@root]`                                                                                        |
+| status                                           | fixed value = `"final"`                                                                                                  |
+| code                                             | `CompoundStatement / code` as described in the XML > FHIR section of [Codeable Concept](../codeable%20concept/README.md) |
+| subject                                          | reference to the mapped [Patient](../patient/README.md)                                                                  |
+| context                                          | reference to the associated [Encounter](../encounters/README.md) (if present)                                            |
+| effectiveDateTime                                | `CompoundStatement / effectiveTime` or else `CompoundStatement / availibiltyTime [@value]`                               |
+| issued                                           | `ehrCompostion / author / time [@value]` <sup>2</sup> or else `EhrExtract / availibilityTime [@value]`                   |
+| performer\[0]                                    | Practitioner referenced in `ObservationStatement / paticipant` <sup>3</sup>                                              |
+| related\[index].type                             | fixed value = `"has-member"`                                                                                             |
+| related\[index].target                           | reference to the child Observation - [Test Result](#Test-Result-(XML-HL7->-JSON-FHIR))                                   |
+
+<details>
+<summary>Example JSON</summary>
+
+```JSON
+
+ {
```

Comment:
Minor, but the indent formatting in this JSON blob looks off by 1.

Reply:
fixed